### PR TITLE
Add Refoss sensor platform

### PR DESCRIPTION
**Pull request by ashionky**
## Proposed change

Add model description 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/116965
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
  - Added support for Sensor category in the Refoss integration.
  - Updated supported platforms to include sensors.

- **Documentation**
  - Revised the supported device models to include additional models and their respective versions.

### CODE REVIEW

File: source/_integrations/refoss.markdown [hunk 1]
```diff
@@ -28,8 +30,12 @@ Integrate Refoss devices into Home Assistant.
 
 ### Supported device model
```
Comment (c0ffeeca7) [inline source/_integrations/refoss.markdown]:
```suggestion
### Supported device models
```

File: source/_integrations/refoss.markdown [hunk 2]
```diff
@@ -28,8 +30,12 @@ Integrate Refoss devices into Home Assistant.
...
 
-- Refoss Smart Wi-Fi Switch, R10
-  
+| model | version            |
```
Comment (c0ffeeca7) [inline source/_integrations/refoss.markdown]:
```suggestion
| Model | Version            |             
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @ashionky :+1:

File: source/_integrations/refoss.markdown [hunk 3]
```diff
@@ -2,14 +2,16 @@
 title: Refoss
 description: Integrate Refoss devices
 ha_category:
+  - Sensor
   - Switch
-ha_release: 2024.1
+ha_release: 2024.6
```
Comment (edenhaus) [inline source/_integrations/refoss.markdown]:
```suggestion
ha_release: 2024.1
```

This value should not updated. It points to the first version, where the integration was added

**Review by edenhaus — CHANGES_REQUESTED** [review]

**Review by edenhaus — APPROVED** [review]

**Pull request merged**